.github/pull_request_template.md

```diff
@@ -1,7 +1,7 @@
 **Description**
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
-Fixes #(issue)
+Resolves #(replace parentheses and the stuff here with the issue number, if there isn't one, create one.) 
```

Comment:
Some clarification on this:
- Explicit instruction on linking an issue with a PR
- In the context of using the same PR template for bugfix and feature PRs, "Resolves" sounds more neutral to both causes than "Fixes"